### PR TITLE
Change controller-runtime pkg/runtime/log to pkg/log

### PR DESCRIPTION
**Pull request by racevedoo**
This includes changes mentioned in #1708 relative to `pkg/runtime/log` deprecations on controller-runtime. Another PR will follow with the `pkg/utils/log` refactor

### CODE REVIEW

**Comment by pebrc** [comment]
Jenkins test this please

**Comment by racevedoo** [comment]
@pebrc I guess jenkins was shut down during the build. could you trigger a rebuild please?
From jenkins console:
image

**Comment by barkbay** [comment]
jenkins test this please

File: pkg/controller/common/certificates/ca_test.go [hunk 1]
```diff
@@ -46,7 +47,9 @@ var (
 )
 
 func init() {
-	logf.SetLogger(logf.ZapLogger(false))
+	logf.SetLogger(crzap.New(func(o *crzap.Options) {
```
Comment (charith-elastic) [inline pkg/controller/common/certificates/ca_test.go]: Looks like we missed this unnecessary logger initialisation during the earlier refactoring. Do you mind getting rid of it?

File: pkg/dev/portforward/pod_forwarder_test.go [hunk 2]
```diff
@@ -17,11 +17,14 @@ import (
...
 
 	"github.com/stretchr/testify/assert"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	crzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
 func init() {
-	logf.SetLogger(logf.ZapLogger(true))
+	logf.SetLogger(crzap.New(func(o *crzap.Options) {
```
Comment (charith-elastic) [inline pkg/dev/portforward/pod_forwarder_test.go]: Logger initialisation unnecessary here as well

File: pkg/utils/test/integration.go [hunk 3]
```diff
@@ -33,7 +34,9 @@ var Config *rest.Config
 // RunWithK8s starts a local Kubernetes server and runs tests in m.
 func RunWithK8s(m *testing.M, crdPath string) {
 	_ = controllerscheme.SetupScheme()
-	logf.SetLogger(logf.ZapLogger(true))
+	logf.SetLogger(crzap.New(func(o *crzap.Options) {
```
Comment (charith-elastic) [inline pkg/utils/test/integration.go]: Ditto for this logger initialisation as well.

File: pkg/controller/common/certificates/ca_test.go (hunk 1 above)
Comment (racevedoo) [reply pkg/controller/common/certificates/ca_test.go]: done!

**Comment by charith-elastic** [comment]
Jenkins test this please

**Pull request merged**